doc/plugins.md

```diff
@@ -87,6 +87,8 @@ See [Creating plugins][create] below.
     — add `target` and `rel` attributes to external links
 *   [`remark-extract-frontmatter`](https://github.com/mrzmmr/remark-extract-frontmatter)
     — store front matter in vfiles
+*   [`remark-fenced-divs`](https://github.com/benabel/remark-fenced-divs)
+    — port of the pandoc extension fenced_divs in remark. (new node types, rehype compatible)
```

Comment:
```suggestion
*   [`remark-fenced-divs`](https://github.com/benabel/remark-fenced-divs)
    — pandoc `fenced_divs` support (new node type, rehype compatible)
```

How about this? Bit shorter; to me, feels like the same message?

Reply:
Ok no problem, I'm not a native english speaker.